examples/pubsub/gcp-pubsub/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"github.com/NYTimes/gizmo/examples/nyt"
+	"github.com/NYTimes/gizmo/pubsub"
+	"github.com/sirupsen/logrus"
+	"github.com/NYTimes/gizmo/pubsub/gcp"
+	"context"
+	"fmt"
```

Comment:
Please regroup the imports to match the Go style guide: https://github.com/golang/go/wiki/CodeReviewComments#imports